src/promptflow/tests/sdk_cli_azure_test/e2etests/test_telemetry.py

```diff
@@ -268,3 +268,24 @@ def test_ci_user_agent(self, cli_perf_monitor_agent) -> None:
             raise e
         finally:
             del os.environ[USER_AGENT]
+
+    def test_duplicate_ua(self):
+        context = OperationContext.get_instance()
+        origin_agent = context.get_user_agent()
+        context.append_user_agent('   ua1 ua2 ua3   ')  # Add fixed UA with extra spaces
+
+        os.environ[USER_AGENT] = '    ua3   ua2  ua1'  # Env configuration ua with extra spaces, duplicate ua.
```

Comment:
Will this part work?

I don't see any extra operation to make user_agent change.

Reply:
This part of the code has been deleted